### PR TITLE
Fix visualization and default model for text detection PP-OCR

### PR DESCRIPTION
**Pull request by ryan1288**
Minor PR to correct the default model path and address issues with visualization. The issue was partially discussed in https://github.com/opencv/opencv_zoo/issues/135#issuecomment-1962323969.

### CODE REVIEW

**Comment by fengyuentau** [comment]
@WanliZhong Please take a look at this PR.

File: models/text_detection_ppocr/demo.cpp [hunk 1]
```diff
@@ -172,7 +172,10 @@ int main(int argc, char** argv)
                 imwrite("result.jpg", originalImage);
             }
             if (viz)
+            {
                 imshow(kWinName, originalImage);
+                waitKey(0);
```
Comment (WanliZhong) [inline models/text_detection_ppocr/demo.cpp:177]: Line 30 has
```c++
 while (waitKey(1) < 0)
```
So I think we don‘t need to add `waitKey(0)`?

Comment (ryan1288) [reply models/text_detection_ppocr/demo.cpp:177]: `while (waitKey(1) < 0)` - I believe this has a timeout of 1 ms before continuing the loop.

It then attempts to go into 
```
cap >> originalImage;
if (originalImage.empty())
{
    if (parser.has("input"))
    {
        cout << "Frame is empty" << endl;
        break;
    }
    else
        continue;
}
```
and reaches `cout << "Frame is empty" << endl;`. On the otherhand, waitKey(0) waits indefinitely, allowing the user to keep the processed image up for viewing.

**Review by WanliZhong — APPROVED** [review]
LGTM👍

**Pull request merged**